tests/conftest.py

```diff
@@ -94,7 +96,25 @@ def app():
 def client(app):
     return TestClient(app)
 
+@pytest.fixture
+def image_path_jpeg():
+	image = "cytomine-org-logo.jpeg"
+	path = "/data/pims/upload_test_jpeg/"
+	return [path, image]
```

Comment:
This is a minor comment, but brackets are unnecessary.  You can simply do `return path, image` which returns a Python tuple

Reply:
Also I'm not sure that "path" and "image" are the best variable names here. While I'm sure I didn't respected the following conventions everywhere, I've tried to use these conventions:
* Keep `image` variable name for image content (not the path or the filename)
* Use `filename` as variable name for values like "myimage.png"
* Use `filepath` as variable name for values like "/a/b/c/myimage.png"
* Use `path` or `x_dir` as variable name for directory path like "/a/b/c"

Reply:
Fixed